docs/supported_models.md

```diff
@@ -271,6 +271,17 @@ Neural Speed supports the following models:
     <td> </td>
     <td> </td>
     <td>Latest</td>
+  </tr>
+    <tr>
+    <td><a href="https://huggingface.co/google/gemma-2b-it" target="_blank" rel="noopener noreferrer">gemma-2b-it </a>,
```

Comment:
there is no structure difference between gemma-2b and gemma-2b-it  right ? 
[gemma-2b](https://huggingface.co/google/gemma-2b)

Reply:
yes, gemma-2b-it is fine tune from gemma-2b, they have no difference.